src/appkit.cs

```diff
@@ -13459,7 +13459,7 @@ interface NSSegmentedCell {
 	}
 
 	[BaseType (typeof (NSControl))]
-	interface NSSlider : NSAccessibilitySlider {
+	interface NSSlider : NSUserInterfaceCompression, NSAccessibilitySlider {
```

Comment:
_The header comment should clarify this and using compression on the slider should work as expected if actually used._

I'm not sure the above comment meant `NSSlider` conforms to `NSUserInterfaceCompression` or just it's item's views are, so compression works (because of them).

note: it be easier with a link to trello in the commit message